tools/peptideshaker/fasta_cli.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="fasta_cli" name="FastaCLI" version="1.0">
+    <description>
+        Appends decoy sequences to FASTA files
+    </description>
+    <macros>
+        <import>macros_basic.xml</import>
+    </macros>
+    <requirements>
+	<requirement type="package" version="@SEARCHGUI_VERSION@">searchgui</requirement>
+	<requirement type="package" version="3.0">zip</requirement>
+    </requirements>
+    <expand macro="stdio" />
+    <command>
+<![CDATA[
+
+        ## #set $temp_stderr = "searchgui_stderr"
+        #set $bin_dir = "bin"
+
+        mkdir output;
+        cwd=`pwd`;
+        export HOME=\$cwd;
```

Comment:
If this is really needed, this should be fixed upstream. Relying on a HOME is not good practice.

Reply:
I think it is not necessary in this tool, but not sure about the others. I will try to check them.